src/main/webapp/WEB-INF/classes/.gitignore

```diff
@@ -0,0 +1 @@
+/META-INF/
```

Comment:
Since there are so many irrelevant commits in this pull request, perhaps make a new pull request? This time, be sure to not include the unrelated commits and files.